rts/Sim/Units/Unit.cpp

```diff
@@ -1514,18 +1514,23 @@ void CUnit::ChangeLos(int losRad, int airRad)
 }
 
 
```

Comment:
I think we need to deprecate ChangeTeam as well in order to completely remove reason. Let's pull capture and anything touching ChangeTeam in so we can add a call without reason and clean up all callers. In the same vein, I noticed 

Net::AIShare is not completely in LUA, same for Capture. We want everything to do with sharing in LUA. 
GiveEverythingTo is not deprecated but does have the SyncedActionFallback.
I see a few more places where it's called, like Builder has its own copy, can we streamline all of this and remove it from the engine?